server/mentions_crawler_flask/responses/__init__.py

```diff
@@ -8,55 +9,65 @@
 _COMPANY_NAMES_TAG = "names"
```

Comment:
This should be equal to "companies" because you changed the corresponding frontend constant to that